snapcraft/plugins/catkin.py

```diff
@@ -245,12 +246,18 @@ def build(self):
 
         self.run(
             ['rm', '-f',
-             'opt/ros/' + self.options.rosversion + '/.catkin',
-             'opt/ros/' + self.options.rosversion + '/.rosinstall',
-             'opt/ros/' + self.options.rosversion + '/setup.sh',
-             'opt/ros/' + self.options.rosversion + '/_setup_util.py'],
+             'opt/ros/' + self.options.rosversion + '/.rosinstall'],
```

Comment:
It shouldn't need to be removed, but can be filtered now that the filters are fixed. Never got around to replacing the rm with proper stage filters.